packages/sitecore-jss-react/src/components/PlaceholderCommon.tsx

```diff
@@ -197,7 +197,15 @@ export class PlaceholderCommon<T extends PlaceholderProps> extends React.Compone
             fields: { ...placeholderFields, ...componentRendering.fields },
           }),
           ...((placeholderParams || componentRendering.params) && {
-            params: { ...placeholderParams, ...componentRendering.params },
+            params: {
+              ...placeholderParams,
+              ...componentRendering.params,
+              // Provide SXA styles
+              ...(componentRendering.params.FieldNames && {
+                styles: `${componentRendering.params.GridParameters || ''} ${componentRendering
+                  .params.Styles || ''}`,
```

Comment:
Might be good to extract this logic to separate function e.g. `getSxaParams`? There's a bit of SXA magic here.

Reply:
Done